Gemfile

```diff
@@ -4,6 +4,7 @@ gem 'rake'
 gem 'mumukit', github: 'mumuki/mumukit', tag: 'v0.4.0'
 gem 'mumukit-inspection', github: 'uqbar-project/mumukit-inspection', branch: 'master'
 gem 'stones-spec', github: 'uqbar-project/stones-spec', branch: 'master'
+gem 'recursive-open-struct', github: 'mumuki/recursive-open-struct', branch: 'feature-use-symbols-for-hash-representation'
```

Comment:
Hmm, I think that it would be a better idea to include activesupport gem, which already has an extension methods to hashes: http://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html#method-i-with_indifferent_access

 .